packages/core/src/link/Link.css

```diff
@@ -6,12 +6,26 @@
   --link-color-focus: "inherit";
   --link-focus-outlineColor: currentColor;
 
-  --link-textDecoration: var(--salt-navigable-textDecoration);
-  --link-textDecoration-hover: var(--salt-navigable-textDecoration-hover);
-
   --link-fontFamily: var(--salt-text-fontFamily);
 }
 
+.saltLink-underlineLink {
```

Comment:
Looks like we choose, link value based on it becoming a suffix for the className ?